sources/HUBViewControllerImplementation.m

```diff
@@ -864,6 +870,7 @@ - (void)createCollectionViewIfNeeded
     collectionView.dataSource = self;
     collectionView.delegate = self;
     
+    [self.view addGestureRecognizer:collectionView.panGestureRecognizer];
```

Comment:
Could this ever cause the gesture to be recognized twice? E.g. once in the UICollectionView, and once in its superview?